packages/firebase_core/firebase_core_platform_interface/test/firebase_options_test.dart

```diff
@@ -23,6 +23,7 @@ void main() {
           projectId: 'projectId');
 
       expect(options1 == options2, isTrue);
+      expect(options1.hashCode == options2.hashCode, isTrue);
```

Comment:
What was the reason to change this to `.hashCode`?

Reply:
Not changed, but addded.
It failed before this PR because original implementation was broken.

Reply:
Oh, my issue! I did not look properly and thought that the line was replaced and not added. Everything fine 👌